api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/ClaimMessageSender.java

```diff
@@ -88,4 +88,14 @@ public void sendUpdateYourAddressLetterMessage(Claim claim) {
         MessagePayload payload = buildLetterPayloadWithAddressOnly(claim, LetterType.UPDATE_YOUR_ADDRESS);
         messageQueueClient.sendMessage(payload, SEND_LETTER);
     }
+
+    public void sendInstantSuccessChildrenMatchLetterMessage(Claim claim) {
```

Comment:
We've got 3 letter methods here now that only vary by `LetterType`, can we refactor into a single method please?

Reply:
Refactored into one method.